### DISCUSSION TITLE
browser.gatherUsageStats

### DISCUSSION THREAD

**Discussion by TheBig-O** [neonwatty/meme-search#16] [Q&A]
In the startup logs, I see the message "Collecting usage statistics. To deactivate, set browser.gatherUsageStats to false."
Where exactly are we supposed to put this command to stop usage stats? After digging around a bit, the only place I found to put it is the dockerfile and rebuild. This seems like something that should be set to "off" as a default. There's no indication of what, exactly is being sent and where.
Please advise.

**Comment by neonwatty** [comment]
looks like this is being done by streamlit - the open source front end we're currently using!   looking around [it seems this is an issue they've had for a while](https://github.com/streamlit/streamlit/issues/7202).

i was not aware of this and will set this to default False with an update this weekend.  looks like it requires the addition of a [config file](https://docs.streamlit.io/develop/api-reference/configuration/config.toml).

**Reply by TheBig-O** [reply]
Thanks for the quick reply and great work on the app! I'll see if I can sort out how to incorporate the config file.

**Reply by neonwatty** [reply]
Just [added a config with gatherUsageStats set to false](https://github.com/neonwatty/meme_search/blob/main/.streamlit/config.toml).   I'm not seeing this message about gatherUsageStats on my end - can you check on yours?  I'm not sure how to write a test for this other than to "trust" the config.

**Reply by TheBig-O** [reply]
Sorry, it's still showing. thanks for trying though.
I added the .streamlit directory and the config file like you showed and it didn't affect it at all.

**Comment by neonwatty** [comment]
Also solved in the new "pro" version of the app!
